XRTK-Core/Packages/com.xrtk.core/Runtime/Providers/CameraSystem/BaseCameraDataProvider.cs

```diff
@@ -345,6 +355,10 @@ private bool SetupTrackingOrigin()
                 // connected, position the camera at the configured default offset.
                 UpdateCameraHeightOffset(defaultHeadHeight);
             }
+#else
```

Comment:
See comment above. This does nothing except adding useless lines of code.

Reply:
Removed